peachjam/templates/peachjam/layouts/document_detail.html

```diff
@@ -421,6 +435,14 @@ <h5>
           <div class="container">{% include 'peachjam/_citations.html' %}</div>
         </div>
       {% endif %}
+      {% if document.case_histories.all|length %}
```

Comment:
It's better to call `.case_histories.exists()`, otherwise it loads all the data, counts the length, and then throws the data away. Using `.exists()` just checks for at least one row in the DB without loading any data.

Reply:
ok. changed. We now load the case histories in from view with prefetched related fields